documentation/meta/documentation/quickstart.md

```diff
@@ -5,50 +5,22 @@ locally.
 
 ## Prerequisites
 
-Refer to the [general setup guide](/general/general_setup.md) for setting up the
-prerequisites. Refer to the 'Docs' column in the
-[requirement matrix](/general/general_setup.md#requirement-matrix) to know what
-you need to run this.
+Follow the [general setup guide](/general/general_setup.md) to set up `ov`.
 
 ## Starting up
 
-1. Ensure you download, install and set up all prerequisites.
-
-2. Clone the repository to your computer. Then switch to the cloned directory.
-   If you're planning to contribute, fork the repo and clone your fork instead.
-
-   ```{note}
-   We recommend cloning with the `--filter=blob:none` flag as it dramatically
-   reduces the filesize and download time by creating a "blobless clone".
-   You can learn more about these [here](https://gist.github.com/leereilly/1f4ea46a01618b6e34ead76f75d0784b).
-   ```
-
-   ```bash
-   git clone --filter=blob:none https://github.com/WordPress/openverse.git # or your fork
-   cd openverse/
-   ```
-
-   If you followed the general setup guide and installed
-   [GitHub CLI](/general/general_setup.md#github-cli), you can clone more simply
-   using the `gh` command.
-
-   ```bash
-   gh repo clone WordPress/openverse -- --filter=blob:none  # or your fork
-   cd openverse/
-   ```
-
-3. Install only the Python dependencies. You do not need to install any Node.js
+1. Install only the Python dependencies. You do not need to install any Node.js
```

Comment:
So nice not to worry about picking the exact dependencies that Node/pnpm needs! 💯

Reply:
We can probably simplify this too, really! This is really written in a context of the old getting started page that had that matrix of different dependencies. If `ov just documentation/install` is treated as the entry point for setting up the docs locally, then we don't really need to specify what the things installed are in this documentation (it'll inevitably drift!). Just to say, I think we can simplify these statements further now :grin: